scene/2d/cpu_particles_2d.cpp

```diff
@@ -33,6 +33,7 @@
 #include "scene/2d/canvas_item.h"
 #include "scene/2d/particles_2d.h"
 #include "scene/resources/particles_material.h"
+#include "core/core_string_names.h"
```

Comment:
Includes should be ordered alphabetically, so this failed `clang-format` style checks.